modules/http/src/main/scala/ch/epfl/bluebrain/nexus/commons/http/HttpClient.scala

```diff
@@ -94,14 +120,14 @@ object HttpClient {
     * @param um an implicit ''FromEntityUnmarshaller[A]''
     * @tparam A the specific type to which the response entity should be unmarshalled into
     */
-  final implicit def withAkkaUnmarshaller[A: Typeable](implicit
+  final implicit def withAkkaUnmarshaller[A: ClassTag](implicit
```

Comment:
what's the reason for this change?

Reply:
There wasn't found the `Typeable[QueryResults[A]]`

Reply:
the `Typeable` type or a typeclass instance for `Typeable[QueryResults[A]]`?